samples/bluetooth/hci_uart_async/app.overlay

```diff
@@ -0,0 +1,31 @@
+/* This is the default app device tree overlay. This file is ignored if
+ * there is a board-specific overlay in `./boards`.
+ *
+ * Most boards define a convenient `&uart0`. It's used here to make the
+ * sample 'just work' automatically for those boards.
```

Comment:
just works

Reply:
You mean like below? I think it sounds worse.

> It's used here to make the sample 'just works' automatically for those boards.

Reply:
Huh, maybe the diff changed when I commented. Agree it makes more sense as-is.